entity-service/src/main/kotlin/com/egm/stellio/entity/authorization/Group.kt

```diff
@@ -0,0 +1,37 @@
+package com.egm.stellio.entity.authorization
+
+import com.egm.stellio.shared.util.AuthContextModel.AUTH_REL_IS_MEMBER_OF
+import com.egm.stellio.shared.util.JsonLdUtils
+import com.egm.stellio.shared.util.JsonLdUtils.NGSILD_NAME_PROPERTY
+import java.net.URI
+
+data class Group(
+    val id: URI,
+    val type: List<String>,
+    var isMember: Boolean? = null,
```

Comment:
what about setting a default value to false instead?

Reply:
Yes it's better

Reply:
I put null to differentiate when we were in admin mode or not

Reply:
and it should be a `val`, not a `var`